test/dist.ini

```diff
@@ -4,4 +4,11 @@ license = MIT
 version = v0.0.0
 copyright_holder = SmartBear Software
 
-[@Basic]
+[@Filter]
+-bundle=@Basic
+-remove=GatherDir
+
+[Git::GatherDir]
+[GatherFile]
+; explicitly add unversioned files
+filename=CHANGELOG.md
```

Comment:
Instead of doing this in the action, could you set the `root` attribute to the parent directory?

https://metacpan.org/pod/Dist::Zilla::Plugin::GatherFile#root

Reply:
I think it may even work:

https://github.com/cucumber/gherkin/blame/main/perl/dist.ini#L37 :smile:

Reply:
Yes, I mean, that silenced the failure, but it didn't end up in the release archive. See https://metacpan.org/release/CUKEBOT/Gherkin-25.0.2/source

Reply:
> could you set the `root` attribute to the parent directory?

this is an interesting approach. let me check that out.